week-1/exercises/random.js

```diff
@@ -6,4 +6,6 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
-// Try logging the value of num several times to build an idea of what the program is doing
+// Try logging the value of num several times to build an idea of what the program is doing - math,random gives a value between 0 and 1, however 0 and 1 are not included. max - min +1 is 100 so a random number multiplied by 100 is anywhere between 0 and 100 but 0 and 100 not included. then add +1 and then 100 is included. math. floor rounds the number. so this function is giving us a random number between 0 and 100
```

Comment:
You've clearly broken down the expression well 👏 
The answer is a little bit off. "math,random gives a value between 0 and 1, however 0 and 1 are not included" -> this statement isn't quite right 
Double check the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) on this and try once again to identify the range of numbers that would be returned. You're not far off!

Reply:
the values would be between 1 and 100.

Reply:
it would be between 1 and 100